third_party/move/move-compiler/src/parser/keywords.rs

```diff
@@ -22,6 +22,7 @@ pub const KEYWORDS: &[&str] = &[
     "module",
     "move",
     "native",
+    "package",
```

Comment:
Please do not make this a keyword, we do not have breaking changes in Move 2.

Instead use a 'soft' keyword, that is interpreting an identifier in a special way in a given context. There are multiple examples for this already in syntax.rs, search e.g. for `"schema"` or check `"enum"` in the enum PR.

Reply:
done